nemo/collections/asr/models/clustering_multi_ch_diarizer.py

```diff
@@ -617,7 +617,7 @@ def _run_clustering(self, embeddings, time_stamps, vad_probs, scale_mapping):
             multiscale_weights=self._diarizer_params.speaker_embeddings.parameters.multiscale_weights,
             multiscale_dict=self.multiscale_args_dict['scale_dict'],
             verbose=self.verbose,
-            vad_threshold=self._diarizer_params.vad.parameters.frame_vad_threshold,
+            vad_threshold=self._diarizer_params.clustering.parameters.frame_vad_threshold,
```

Comment:
what's the motivation of this change?

Reply:
I think that maybe it is good to have stricter VAD for embedding extraction ? 
Because right now the speaker counting is off. Maybe we don't need that because MSDD will refine the predictions nonetheless.

Reply:
It seems that changing the name from `_diarizer_params.vad.parameters.frame_vad_threshold` to `_diarizer_params.clustering.parameters.frame_vad_threshold` won't affect anything, as `frame_vad_threshold` is only used here....? @tango4j am I understanding it correctly?